### DISCUSSION TITLE
Input / output Controls

### DISCUSSION THREAD

**Discussion by Vau9hn71** [frangoteam/FUXA#1257] [Q&A]
Is it possible to add the same properties for input controls as it is for output controls. Output controls have a 'value', 'unit' and 'format digit' property, whereas input controls only have a value property. 

Also, the input control seems to have a border that is not editable?

It would be nice if an output control can also be selected to have an input function based on authorization.

Thanks,
Vaughn

**Comment by unocelli** [comment]
Hi, I would add it as a features in the issues section, maybe even separate.
